sources/packages/api/src/services/institution/institution.service.ts

```diff
@@ -506,4 +506,34 @@ export class InstitutionService extends RecordDataModelService<Institution> {
     }
     return searchQuery.getMany();
   }
+
+  /**
+   * Get the institution by ID.
+   * @param institutionId Institution id.
+   * @returns Location retrieved, if found, otherwise returns null.
+   */
+  async getAESTInstitutionDetailById(
+    institutionId: number,
+  ): Promise<Institution> {
+    return this.repo
+      .createQueryBuilder("institution")
+      .innerJoinAndSelect("institution.institutionType", "institutionType")
```

Comment:
Please change the query to return only what is needed, as we talked about in the previous meeting.

Reply:
There is no need to select the required columns only, for this query, as we are using all the objects in the institution object.

Reply:
I was talking about the specific line that I added to the comment.
```
.innerJoinAndSelect("institution.institutionType", "institutionType")
```
The only property being used is _institutionDetail.institutionType.name_.

Reply:
ho my bad 👍